File: projetos/crud-spring-reactive/src/main/java/study/crudspring/pessoa/PessoaEntity.java
package study.crudspring.pessoa;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

import java.time.LocalDate;

@Table("pessoas")
public class PessoaEntity {

    @Id
    private Long id;

    private String apelido;
    private String nome;
    private LocalDate nascimento;

    public void setId(Long id) {
        this.id = id;
    }

    public Long getId() {
        return id;
    }

    public String getApelido() {
        return apelido;
    }

    public void setApelido(String apelido) {
        this.apelido = apelido;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public LocalDate getNascimento() {
        return nascimento;
    }

    public void setNascimento(LocalDate nascimento) {
        this.nascimento = nascimento;
    }
}

File: projetos/nestjs-app-deno/src/pessoa/pessoa.entity.ts
import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';

@Entity('pessoas')
export class Pessoa {
  @PrimaryGeneratedColumn()
  id?: number;

  @Column()
  nome?: string;

  @Column()
  apelido?: string;

  @Column({type: 'date'})
  nascimento?: Date;
}